builtin/mainmenu/tab_mods.lua

```diff
@@ -31,16 +31,9 @@ local function get_formspec(tabview, name, tabdata)
 		"tablecolumns[color;tree;text]" ..
 		"table[0,0.25;5.1,5;modlist;" ..
 		modmgr.render_modlist(modmgr.global_mods) ..
-		";" .. tabdata.selected_mod .. "]"
-
-	retval = retval ..
---		"label[0.8,4.2;" .. fgettext("Add mod:") .. "]" ..
---		TODO Disabled due to upcoming release 0.4.8 and irrlicht messing up localization
---		"button[0.75,4.85;1.8,0.5;btn_mod_mgr_install_local;".. fgettext("Local install") .. "]" ..
-
---		TODO Disabled due to service being offline, and not likely to come online again, in this form
---		"button[0,4.85;5.25,0.5;btn_modstore;".. fgettext("Online mod repository") .. "]"
-		""
+		";" .. tabdata.selected_mod .. "]" ..
+		"button[10.2,0;1.8,0.5;btn_mod_mgr_install_local;".. fgettext("Install mod") .. "]" ..
```

Comment:
`fgettext` already does this AFAIK (`core.formspec_escape` isn't used _anywhere_ in the main menu but rather fgettext so I assume it does).